core/server/services/auth/auth.js

```diff
@@ -0,0 +1,5 @@
+const authenticate = require('./authenticate'),
+    authorize = require('./authorize');
+
+module.exports.contentAPI = [authenticate.authenticateContentAPI, authorize.authorizeContentAPI];
+module.exports.adminAPI = [authenticate.authenticateAdminAPI, authorize.authorizeAdminAPI];
```

Comment:
In general: I wonder if the general auth service should know about a "content" and an "admin api" 🤔Maybe it could just offer a specific interface?

e.g. instead of authorizeAdminAPI -> authorizeSession 
e.g. `services.auth.session.authorize`

Whatever you like. These are just suggestions.

> core/server/services/auth/auth.js

Also: `auth/auth` and `services.auth.auth.X` 🤔

---

I guess the design here can be improved. Let me know if you wanne talk about it :)

Reply:
Yeah the idea here was to mirror the `authorizeUserPublicApi` stuff, to make it simple for you to integrate with the apis, but still expose the lower level stuff for when you wanted to use that. Happy to change this though :+1: 

As for `auth/auth`, I figured we have `auth/authenticate` and `auth/authorize` so `auth/auth` is the mix of both of them - it is weird though - agreed.

Reply:
> appy to change this though 👍

Let's talk about this in slack later?